app/models/concerns/oregon_digital/image_metadata.rb

```diff
@@ -37,6 +36,10 @@ module ImageMetadata
       end
 
       property :width, predicate: ::RDF::Vocab::EXIF.width, multiple: false
+
+      define_singleton_method :image_properties do
+        (properties.select { |_k, v| v.class_name.nil? }.keys - initial_properties)
```

Comment:
The assumption here is that any property with a `class_name` must be a controlled vocabulary.. it would be better if this checked for a class_name and that the class name doesn't include `ControlledVocabularies`?